[2022-08-06 19:36:06,949] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 19:36:06,964] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 19:36:06,964] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:06,964] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:36:06,964] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:06,984] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-11 00:00:00+00:00
[2022-08-06 19:36:06,990] {standard_task_runner.py:52} INFO - Started process 222 to run task
[2022-08-06 19:36:06,994] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-11T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3h5j4ap2', '--error-file', '/tmp/tmpl6_vk0yb']
[2022-08-06 19:36:06,994] {standard_task_runner.py:80} INFO - Job 45: Subtask second_task
[2022-08-06 19:36:07,074] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:36:07,171] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2022-08-06 19:36:07,172] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:36:07,172] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:36:07,185] {subprocess.py:85} INFO - Output:
[2022-08-06 19:36:07,187] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:36:07,187] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:36:07,229] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210811T000000, start_date=20220806T193606, end_date=20220806T193607
[2022-08-06 19:36:07,285] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:36:07,318] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:02,422] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 21:50:02,442] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 21:50:02,442] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:02,442] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:02,442] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:02,462] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-11 00:00:00+00:00
[2022-08-06 21:50:02,469] {standard_task_runner.py:52} INFO - Started process 411 to run task
[2022-08-06 21:50:02,472] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-11T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpysdw0ve2', '--error-file', '/tmp/tmpitllfdpk']
[2022-08-06 21:50:02,473] {standard_task_runner.py:80} INFO - Job 43: Subtask second_task
[2022-08-06 21:50:02,567] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:02,662] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2022-08-06 21:50:02,663] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:02,664] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:02,680] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:02,682] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:02,682] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:02,722] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210811T000000, start_date=20220806T215002, end_date=20220806T215002
[2022-08-06 21:50:02,764] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:02,804] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:55:33,172] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 21:55:33,188] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 21:55:33,188] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:55:33,188] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:55:33,188] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:55:33,206] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-11 00:00:00+00:00
[2022-08-06 21:55:33,214] {standard_task_runner.py:52} INFO - Started process 209 to run task
[2022-08-06 21:55:33,218] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-11T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpphs13rir', '--error-file', '/tmp/tmpz578r8yp']
[2022-08-06 21:55:33,219] {standard_task_runner.py:80} INFO - Job 44: Subtask second_task
[2022-08-06 21:55:33,328] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:55:33,436] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2022-08-06 21:55:33,437] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:55:33,438] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:55:33,458] {subprocess.py:85} INFO - Output:
[2022-08-06 21:55:33,461] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:55:33,462] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:55:33,535] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210811T000000, start_date=20220806T215533, end_date=20220806T215533
[2022-08-06 21:55:33,590] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:55:33,657] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:10:32,095] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 22:10:32,108] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 22:10:32,108] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:10:32,109] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:10:32,109] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:10:32,127] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-11 00:00:00+00:00
[2022-08-06 22:10:32,134] {standard_task_runner.py:52} INFO - Started process 207 to run task
[2022-08-06 22:10:32,139] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-11T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3n77gc9p', '--error-file', '/tmp/tmppvvu4ghr']
[2022-08-06 22:10:32,139] {standard_task_runner.py:80} INFO - Job 41: Subtask second_task
[2022-08-06 22:10:32,235] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:10:32,347] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2022-08-06 22:10:32,348] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:10:32,348] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:10:32,365] {subprocess.py:85} INFO - Output:
[2022-08-06 22:10:32,367] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:10:32,367] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:10:32,406] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210811T000000, start_date=20220806T221032, end_date=20220806T221032
[2022-08-06 22:10:32,435] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:10:32,492] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:18:19,805] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 22:18:19,818] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 22:18:19,818] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:18:19,818] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:18:19,818] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:18:19,838] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-11 00:00:00+00:00
[2022-08-06 22:18:19,846] {standard_task_runner.py:52} INFO - Started process 230 to run task
[2022-08-06 22:18:19,850] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-11T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3vv82clb', '--error-file', '/tmp/tmpwhtw9dlu']
[2022-08-06 22:18:19,850] {standard_task_runner.py:80} INFO - Job 43: Subtask second_task
[2022-08-06 22:18:19,934] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:18:20,038] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2022-08-06 22:18:20,039] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:18:20,040] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:18:20,053] {subprocess.py:85} INFO - Output:
[2022-08-06 22:18:20,055] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:18:20,055] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:18:20,086] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210811T000000, start_date=20220806T221819, end_date=20220806T221820
[2022-08-06 22:18:20,141] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:18:20,172] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:21:53,236] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 22:21:53,250] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [queued]>
[2022-08-06 22:21:53,250] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:21:53,250] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:21:53,250] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:21:53,268] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-08-11 00:00:00+00:00
[2022-08-06 22:21:53,275] {standard_task_runner.py:52} INFO - Started process 250 to run task
[2022-08-06 22:21:53,279] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-11T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp05bisc7c', '--error-file', '/tmp/tmp1vopawv_']
[2022-08-06 22:21:53,280] {standard_task_runner.py:80} INFO - Job 41: Subtask second_task
[2022-08-06 22:21:53,356] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-11T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:21:53,470] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-11T00:00:00+00:00
[2022-08-06 22:21:53,471] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:21:53,472] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:21:53,489] {subprocess.py:85} INFO - Output:
[2022-08-06 22:21:53,491] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:21:53,491] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:21:53,530] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210811T000000, start_date=20220806T222153, end_date=20220806T222153
[2022-08-06 22:21:53,611] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:21:53,651] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
